packages/interactivity/src/directives.js

```diff
@@ -102,17 +102,24 @@ export default () => {
 			const inheritedValue = useContext( inheritedContext );
 			const currentValue = useRef( deepSignal( {} ) );
 			const passedValues = context.map( ( { value } ) => value );
-
-			currentValue.current = useMemo( () => {
-				const newValue = context
-					.map( ( c ) => deepSignal( { [ c.namespace ]: c.value } ) )
-					.reduceRight( mergeDeepSignals );
-
-				mergeDeepSignals( newValue, inheritedValue );
-				mergeDeepSignals( currentValue.current, newValue, true );
-				return currentValue.current;
-			}, [ inheritedValue, ...passedValues ] );
-
+			try {
+				const { namespace, value } = context.find(
+					( { suffix } ) => suffix === 'default'
+				);
+				currentValue.current = useMemo( () => {
+					const newValue = deepSignal( {
+						[ namespace ]: value,
+					} );
+					mergeDeepSignals( newValue, inheritedValue );
+					mergeDeepSignals( currentValue.current, newValue, true );
+					return currentValue.current;
+				}, [ inheritedValue, ...passedValues ] );
```

Comment:
I don't expect `useMemo()` to throw an error. Moving this outside of the `try { ... } catch` would be better.

Reply:
However, if this is intentional, then we need to include a test for that case, don't we? 🤔

Reply:
I added previously cause there was errors of trying to map an undefined variable.
As we are now removing:
```
.map( ( c ) => deepSignal( { [ c.namespace ]: c.value } ) )
					.reduceRight( mergeDeepSignals );
```
That possible error has gone. Now it works fine with two contexts. As you mentioned to add to the test.